src/muya/lib/contentState/imageCtrl.js

```diff
@@ -1,5 +1,6 @@
 import { URL_REG, DATA_URL_REG } from '../config'
 import { correctImageSrc } from '../utils/getImageInfo'
+import { shell } from 'electron'
```

Comment:
Files in `src/muya/*` must be plain JavaScript and Node/Electron APIs are forbidden. Please use a callback, like for image resolving via Muya's `options`.

Reply:
Thanks for the review. I will fix it.

Reply:
In the last commit, I removed the Electron API calls from Muya.